src/components/backend-ai-registry-list.ts

```diff
@@ -565,42 +655,38 @@ class BackendAIRegistryList extends BackendAIPage {
   /**
    * Render control units - delete (delete registry), refresh (rescan image).
    *
-   * @param {Element} root - the row details content DOM element
-   * @param {Element} column - the column element that controls the state of the host element
-   * @param {Object} rowData - the object with the properties related with the rendered item
+   * @param {element} root - the row details content DOM element
+   * @param {element} column - the column element that controls the state of the host element
+   * @param {object} rowData - the object with the properties related with the rendered item
    * */
-  _controlsRenderer(root, column, rowData) {
+  private _controlsRenderer(root, column, rowData) {
     render(
       html`
-        <div
-          icon="settings"
-          id="controls"
-          class="layout horizontal flex center"
-        >
+        <div icon="settings" id="controls" class="layout horizontal flex center">
           <wl-button fab flat inverted
               class="fg blue"
               @click=${() => {
```

Comment:
@dan-2ee 
Former indentation degrades readability. It might be originated from current linting rules.
Let's double-check whether the indentation is correct after auto-fixing via the linting command.
If something went wrong, please revert to the original indentation.